website/src/documentation/io/testing.md

```diff
@@ -266,10 +169,9 @@ If you're using Kubernetes scripts to host data stores, make sure you can connec
     1.  JDBC: `kubectl delete -f .test-infra/kubernetes/postgres/postgres.yml`
     1.  Elasticsearch: `bash .test-infra/kubernetes/elasticsearch/teardown.sh`
 
-##### integrationTest Task {#integration-test-task}
-
-Since `performanceTest` task involved running PerfkitBenchmarker, we can't use it to run the tests manually. For such purposes a more "low-level" task called `integrationTest` was introduced.  
+#### Running the test {#running-the-test}
```

Comment:
The section here and below `Running Integration Tests on Pull Requests {#running-on-pull-requests}` seems two ways of running the tests. Should they be in same level? Also this seems focus on running tests with local Gradle, but not very clear to me.

Reply:
Thanks @markflyhigh! I don't think they should. This paragraph describes the way to run a particular test alone. It's a part of a more highlevel paragraph about running any test on your machine.  The other describes the ability of phrase triggering using Beam's CI in general.

I added some sentences to make this more clear. Let me know if this was helpful. :)